manage_people/templates/manage_people/results_list.html

```diff
@@ -3,7 +3,15 @@
 {% load manage_people_tags %}
 {% load static %}
 
-{% block breadcrumbs %}<a href="{% url 'manage_people:user_form' %}">Manage People</a> <i class="fa fa-chevron-right"></i> Add People{% endblock breadcrumbs %}
+{% block breadcrumbs %}
+<nav>
+    <h3>
+        <a href="{% url 'dashboard_course' %}">Manage Course</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:user_form' %}">Manage People</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:find_user' %}">Search for people</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;Add People
```

Comment:
I see what you mean. There is a result page that we could have the same breadcrumb as the search page. There is a link on that page people can use to return to the search page.